src/ert/scheduler/realization_state.py

```diff
@@ -123,17 +123,18 @@ def on_enter_RUNNING(self) -> None:
         self.start_time = datetime.datetime.now()
 
     def on_enter_EXIT(self) -> None:
-        self.realization.run_arg.ensemble_storage.state_map[
-            self.realization.run_arg.iens
-        ] = RealizationStorageState.LOAD_FAILURE
-
         if self.retries_left > 0:
             self.retry()
             self.retries_left -= 1
         else:
+            self.realization.run_arg.ensemble_storage.state_map[
```

Comment:
Shouldn't we change state to `DO_KILL` in here?

Reply:
No, we don't need to kill a realization that has already exited.